android/src/main/java/io/grpc/android/AndroidChannelBuilder.java

```diff
@@ -73,18 +73,43 @@
 
   @Nullable private Context context;
 
-  public static final AndroidChannelBuilder forTarget(String target) {
+  /**
+   * Creates a new builder with the given target string that will be resolved by
+   * {@link io.grpc.NameResolver}.
+   */
+  public static AndroidChannelBuilder forTarget(String target) {
     return new AndroidChannelBuilder(target);
   }
 
+  /**
+   * Creates a new builder with the given host and port.
+   */
   public static AndroidChannelBuilder forAddress(String name, int port) {
     return forTarget(GrpcUtil.authorityFromHostAndPort(name, port));
   }
 
+  /**
+   * Creates a new builder, which delegates to the given ManagedChannelBuilder.
+   *
+   * @deprecated Use {@link #usingBuilder(ManagedChannelBuilder)} instead.
+   */
+  @Deprecated
   public static AndroidChannelBuilder fromBuilder(ManagedChannelBuilder<?> builder) {
     return new AndroidChannelBuilder(builder);
```

Comment:
Let it explicitly call `usingBuilder()`.

Reply:
Done.